assets/css/main.css

```diff
@@ -0,0 +1,211 @@
+    max-width: 1440px;
+    margin: 0 auto;
+}
+
+body a {
+    text-decoration: none;
+    color: white;
+}
+
+#main-bar {
+    background-image: url("../image/your_verse_hero.jpg");
+    background-position: bottom 0px left 0px;
+    background-repeat: no-repeat;
+    background-size: auto;
+    text-align: center;
+    color: white;
+    font-weight: 100;
+    height: 69vh;
+    max-width: 1440px;
+    min-width: 1024px;
+    min-height: 550px;
+}
+
+@media screen and (max-width: 840px) {
+    #main-bar {
+        height: 100vh;
+        width: 820px;
+        background-position: bottom 0px left auto;
+    }
+    #nav-bar,
+    #promote-4 {
+        display: none;
+    }
+}
+
+#nav-bar {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    background: linear-gradient( rgb(153, 149, 149) 0%, rgb(110, 110, 110) 45%, rgb(80, 80, 80) 50%, rgb(120, 120, 120) 98%);
+    border-radius: 3px;
+    box-shadow: 0.5px 0.5px 1px 0.2px rgb(58, 57, 57);
+    height: 36px;
+    width: 980px;
+    margin: 17px auto;
+    font-size: 12px;
+    font-weight: 200;
+}
+
+
+/* nav bar section  */
+
+#nav-bar li {
+    width: 70%;
+    line-height: 36px;
+    vertical-align: middle;
+    border-right: 0.02px rgb(94, 92, 92) solid;
+    box-shadow: 0.5px 0 3px 0 rgba(255, 255, 255, 0.2);
+}
+
+#nav-bar li:first-child {
+    font-size: 23px;
+}
+
+#nav-bar li:hover {
+    background: rgb(66, 65, 65);
+    text-decoration: none;
+    cursor: pointer;
+}
+
+#nav-bar li {
+    width: 80%;
+    margin: 0 10px;
+}
+
+#nav-bar li input {
+    position: relative;
+    width: 126px;
+    height: 22px;
+    border-radius: 12px;
+    border: 1px solid transparent;
+    border-bottom: 0.5px solid rgb(156, 155, 155);
+    box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.758);
+    background-color: rgba(255, 255, 255, 0.096);
+    background: url("../image/search-icon.png") top 5px left 7px no-repeat;
+    background-size: 12px;
+    padding: 1px 0 1px 23px;
+    transition: all 0.3s ease-in-out;
+}
+
+#nav-bar li input:focus {
+    background-color: white !important;
+    background: url("../image/search-icon-black.png") top 3px left 4px no-repeat;
+    background-size: 17px;
+    width: 170px;
+    outline: none;
+    border: none;
+}
+
+
+/* main bar text  */
+
+#main-text {
+    font-size: 32px;
+}
+
+#main-text h1 {
+    font-size: 2em;
+    margin-top: 160px;
+    margin-bottom: 20px;
+}
+
+#main-text p:first-of-type {
+    font-size: 1em;
+    line-height: 1.4em;
+    width: 800px;
+    margin: 25px auto;
+}
+
+#main-text p:nth-of-type(2) {
+    font-size: 0.8em;
+    width: 600px;
+    margin: 25px auto;
+}
+
+#main-text p:nth-of-type(2) i {
+    font-size: 1em;
+    margin-left: 6px;
+}
+
+#main-text p a:hover {
+    text-decoration: underline;
+}
+
+#promote-bar {
+    display: grid;
+    grid-template-columns: repeat(4, auto);
+    justify-content: start;
+    text-align: center;
+    margin-top: 2px;
+    min-width: 1024px;
```

Comment:
Is there a reason why this is having this property `min-width: 1024px;`?
Currently, the footer is not aligned with the main content, see illustration below. :point_down: 

![apple footer not aligned](https://user-images.githubusercontent.com/36057474/65377994-5deee880-dcaa-11e9-8a47-d4433775c94f.gif)

#### Please, consider deleting this property.